### ISSUE TITLE
Models get anchored to draft horse stallions

### ISSUE THREAD

**Issue by TheElemento158** [capnkirok/animania#440]
### Expected Behavior
Models stay attached to the appropriate entity

### Actual Behavior
Models are anchored to a single mob, so far seems to only be with draft horse stallions, sometimes translated to under entity and flipped vertically and the models for doors, villagers, mobs, all seemed to be floating in space, anchored to a single mob and rotating and moving as it did

In the screenshot I attached a chicken's model is floating in the air, and the nearby horse is what it's anchored to. I assure you that is not actually where the chicken is

### Steps to Reproduce
I was on a superflat world on a heavily modded custom pack and it started happening, not sure when or how it started, but it didn't take long to occur

### Version of Minecraft, version of Animania, Single Player or Server
1.12.2, 1.4.1, singleplayer

### Screenshots encouraged
![image](https://user-images.githubusercontent.com/8229720/32989683-efd7263c-cccf-11e7-8555-8bbab5bdb7e1.png)

**Comment by TheElemento158** [comment]
I'd like to also add that sometimes the horse model gets translated under the player as well or disappears

**Comment by capnkirok** [comment]
Wha...? How are you attaching a chicken to a Draft Horse? You must be using a mod of some kind to do this?

**Comment by TheElemento158** [comment]
It isn't attached, it's a visual bug, the chicken isn't actually there. Where the chicken actually is is around five blocks away. It kind of flickers back and forth with anywhere from one to six models getting attached to the horse a fixed distance away. Glitches out the GUI while it flickers as well. When the horse turns the models turn with it, sometimes the horse kind of "comes apart" although it doesn't happen as frequently as the upside-down thing

**Comment by capnkirok** [comment]
Yeah, this isn't Animania. Animania is fine in many, many other packs. It's a mod you've got in the pack that's interacting with it in a bad way. You'll have to post your modlist. I'm guessing some kind of visual mod or animal mod.

**Comment by TheElemento158** [comment]
Still trying to figure out what's causing the bug, but the red screen that happens in #269 also happens with my game, and goes away after another punch

**Comment by TheElemento158** [comment]
Here is my mod list: https://pastebin.com/Q9W6S0ba

In addition, when Animania is added to the following packs:
-Modern Skyblock --bug appears
-All The Mods 3 --bug does not appear
-Direwolf20 1.12 --bug does not appear

**Comment by capnkirok** [comment]
Wow, that’s a ton of (great) mods. I won’t have time to test right away, but my guess is Googly Eyes, as someone just mentioned weird behavior in passing last week. See if that does anything...

**Comment by TheElemento158** [comment]
Huh, that fixed it. I guess there's something with how GooglyEyes renders that bugs out Animania mobs in particular

**Comment by capnkirok** [comment]
I will talk to iChun and see what's happening. Going to rename this.

**Comment by iChun** [comment]
Was there anything printing out to the console whilst the rendering glitches were flickering?

**Comment by TheElemento158** [comment]
The log had this in it while it was happening: https://pastebin.com/EUFJ0CwM

It kept repeating for hundreds of blocks even though I was only in the world for less than a minute

**Comment by capnkirok** [comment]
@iChun not sure if it matters or not, but there are some non-standard naming of model pieces in animania (e.g., "head" is sometimes "HeadNode" )...

**Comment by iChun** [comment]
It's just a null model reference causing the problem. It's an iChunUtil
fix. I'll push it out when I can. Mid exam season at the moment.

**Comment by iChun** [comment]
The rendering issues should be fixed in the next update of iChunUtil.
